apidoc/Titanium/UI/iOS/PreviewContext.yml

```diff
@@ -171,12 +232,19 @@ examples:
         var context = Ti.UI.iOS.createPreviewContext({
             preview: previewView,
             actions: actions, // Can have both Ti.UI.iOS.PreviewAction + Ti.UI.iOS.PreviewActionGroup
-            contentHeight: 300, // When unspecified, we use the available height
-            pop: function(e) { // Called after popping the preview
-                Ti.API.warn("sectionIndex: " + e.sectionIndex); // Index of the ListView section
-                Ti.API.warn("itemIndex: " + e.itemIndex); // Index of the ListView row
-                detailWindow.open();
-            }
+            contentHeight: 300 // When unspecified, we use the available height
+        });
+
+        // Fired after peeking the preview
+        // Use this event to configure the preview depending on the sectionIndex / itemIndex
+        context.addEventListener("peek", function(e) {
```

Comment:
The docs sample are supposed to be as simple as possible. If users want more detailed examples, they normally check out the sample apps or just try out their own ideas